soweego/importer/imdb_dump_extractor.py

```diff
@@ -0,0 +1,390 @@
+                    continue
+
+                professions = professions.split(",")
+
+                # each person can be added to multiple tables in the DB,
+                # each table stands for a profession
+                types_of_entities = []
+
+                if "actor" in professions or "actress" in professions:
+                    self.n_actors += 1
+                    types_of_entities.append(imdb_entity.ImdbActorEntity())
+
+                if "director" in professions:
+                    self.n_directors += 1
+                    types_of_entities.append(imdb_entity.ImdbDirectorEntity())
+
+                if "producer" in professions:
+                    self.n_producers += 1
+                    types_of_entities.append(imdb_entity.ImdbProducerEntity())
+
+                if any(prof in ["sound_department", "composer",
+                                "music_department", "soundtrack"]
+                       for prof in professions):
+                    self.n_musicians += 1
+                    types_of_entities.append(imdb_entity.ImdbMusicianEntity())
+
+                if "writer" in professions:
+                    self.n_writers += 1
+                    types_of_entities.append(imdb_entity.ImdbWriterEntity())
+
+                # if the only profession a person has is `miscellaneous` then we
+                # add it to all tables
+                if professions == "miscellaneous":
+                    self.n_misc += 1
+                    types_of_entities = [
+                        imdb_entity.ImdbActorEntity(),
+                        imdb_entity.ImdbDirectorEntity(),
+                        imdb_entity.ImdbMusicianEntity(),
+                        imdb_entity.ImdbProducerEntity(),
+                        imdb_entity.ImdbWriterEntity(),
+                    ]
+
+                # add person to every matching table
+                for etype in types_of_entities:
+                    self._populate_person(etype, person_info, session)
+
+                # if person is known for any movies then add these to the
+                # databse as well
+                if person_info.get("knownForTitles"):
+                    self.n_person_movie_links += 1
+                    self._populate_person_movie_relations(person_info, session)
+
+                # commit results to the database every `_sqlalchemy_commit_every` loops
+                if self.n_persons % self._sqlalchemy_commit_every == 0:
+                    session.commit()
+
+                self.n_persons += 1
+
+            # finally commit remaining entities
+            session.commit()
+
+        end = datetime.datetime.now()
+        LOGGER.info("Person import completed in %s. "
+                    "Total people imported: %d - "
+                    "Actors: %d - Directors: %d - Musicians: %d - "
+                    "Producers: %d - Writers: %d - Misc: %d",
+                    end - start, self.n_persons, self.n_actors,
```

Comment:
Did you forget to set `start` again for this dump?

Reply:
Good catch! This has been fixed in a8b1c65902c8cf28169db943fe534bcf5c7b840c